frontend/src/scenes/experiments/utils.test.ts

```diff
@@ -96,7 +96,12 @@ describe('utils', () => {
                 ],
             },
             groups: [
-                { properties: [], rollout_percentage: 100 },
+                {
+                    properties: [],
+                    rollout_percentage: 100,
+                    description:
+                        'Added by shipping experiment variant - enables 100% rollout of selected variant to all users',
```

Comment:
The grammar feels a bit clumsy to me.

```suggestion
                        'Added automatically when the experiment variant was shipped',
```